doc/input-output-reference/src/overview/group-airflow-network.tex

```diff
@@ -541,7 +541,7 @@ \subsubsection{Field: Occupant Ventilation Control Name}\label{field-occupant-ve
 
 \subsection{AirflowNetwork:MultiZone:Surface}\label{airflownetworkmultizonesurface}
 
-The AirflowNetwork:MultiZone:Surface object specifies the properties of a surface ``linkage'' through which air flows. This linkage is always associated with a heat transfer surface (wall, roof, floor, or a ceiling) or subsurface (door, glass door, or window) with both faces exposed to air. The linkage specifies two connected nodes: two zone nodes defined in \hyperref[airflownetworkmultizonezone]{AirflowNetwork:MultiZone:Zone} objects based on inside and outside face environment for an interior surface, or a zone node defined in an \hyperref[airflownetworkmultizonezone]{AirflowNetwork:MultiZone:Zone} object based on inside face environment and an external node defined in an \hyperref[airflownetworkmultizoneexternalnode]{AirflowNetwork:MultiZone:ExternalNode} object for an exterior surface. The associated leakage component for this surface can be a crack (or surface effective leakage area) in an exterior or interior heat transfer surface or subsurface, or an exterior or interior window, door or glass door (heat transfer subsurface) that can be opened to allow air flow. The allowed surface air leakage components are:
+The AirflowNetwork:MultiZone:Surface object specifies the properties of a surface ``linkage'' through which air flows. This linkage is always associated with a building surface (wall, roof, floor, or a ceiling) or subsurface (door, glass door, or window) with both faces exposed to air. The linkage specifies two connected nodes: two zone nodes defined in \hyperref[airflownetworkmultizonezone]{AirflowNetwork:MultiZone:Zone} objects based on inside and outside face environment for an interior surface, or a zone node defined in an \hyperref[airflownetworkmultizonezone]{AirflowNetwork:MultiZone:Zone} object based on inside face environment and an external node defined in an \hyperref[airflownetworkmultizoneexternalnode]{AirflowNetwork:MultiZone:ExternalNode} object for an exterior surface. The associated leakage component for this surface can be a crack (or surface effective leakage area) in an exterior or interior heat transfer surface or subsurface, or an exterior or interior window, door or glass door (heat transfer subsurface) that can be opened to allow air flow. The allowed surface air leakage components are:
```

Comment:
Mention air boundary surface options and limitations in various places for AirflowNetwork:MultiZone:Surface.